tests/framework/installer/ceph_installer.go

```diff
@@ -265,7 +265,12 @@ func (h *CephInstaller) CreateCephCluster() error {
 }
 
 func (h *CephInstaller) waitForCluster() error {
-	if err := h.k8shelper.WaitForPodCount("app=rook-ceph-mon", h.settings.Namespace, h.settings.Mons); err != nil {
+	monWaitLabel := "app=rook-ceph-mon,mon_daemon=true"
+	if h.Manifests.Settings().RookVersion == Version1_14 {
+		// TODO: Remove this when upgrade test is from v1.15 since v1.14 does not have the mon_daemon label
+		monWaitLabel = "app=rook-ceph-mon"
```

Comment:
probably, the upgrade test failed due to this?

Reply:
Yes, looking into it...